quiche/src/tests.rs

```diff
@@ -0,0 +1,9848 @@
+// Copyright (C) 2018-2019, Cloudflare, Inc.
```

Comment:
Is there potential for breaking this down further?  Possibly in followup PRs?

Reply:
Yep, 9k lines is too many, but I want to concentrate on production code (e.g. `lib.rs` is still 9k) over the testing code so its lower priority for now imo.